apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/ThrowableAssertionsSpec.kt

```diff
@@ -9,15 +9,31 @@ import ch.tutteli.atrium.specs.property
 class ThrowableAssertionsSpec : ch.tutteli.atrium.specs.integration.ThrowableAssertionsSpec(
     property<Throwable, String>(Expect<Throwable>::message),
     fun1<Throwable, Expect<String>.() -> Unit>(Expect<Throwable>::message),
-    fun2(Expect<Throwable>::messageContains)
+    fun2(Expect<Throwable>::messageContains),
+    "cause" to Companion::causeFeature,
+    "cause" to Companion::cause
 ) {
 
+    companion object {
+
+        @Suppress("RemoveExplicitTypeArguments")
+        private fun causeFeature(expect: Expect<out Throwable>): Expect<IllegalArgumentException> = expect.cause<IllegalArgumentException>()
+
+        @Suppress("RemoveExplicitTypeArguments")
+        private fun cause(expect: Expect<out Throwable>, assertionCreator: Expect<IllegalArgumentException>.() -> Unit) =
+            expect.cause<IllegalArgumentException>(assertionCreator)
+    }
+
     @Suppress("unused", "UNUSED_VALUE")
     private fun ambiguityTest() {
         var a1: Expect<Throwable> = notImplemented()
 
         a1.message
         a1 = a1.message {}
         a1 = a1.messageContains("asdf")
+
+        var a2: Expect<ClassCastException> = notImplemented()
```

Comment:
oho... seems like the type of `a1` was actually not very well chosen by me (in contrast to what I wrote in the comment, you would not have realised that cause did not work for other types).
Could you do me a favour and use `Expect<ClassCastException>` for `a1`? You can then also remove `a2`